main/src/main/scala/sbt/internal/LMCoursier.scala

```diff
@@ -15,9 +15,14 @@ import sbt.librarymanagement._
 import Keys._
 import sbt.internal.librarymanagement.{ CoursierArtifactsTasks, CoursierInputsTasks }
 import sbt.util.Logger
+import sbt.io.syntax._
 
 private[sbt] object LMCoursier {
-  def defaultCacheLocation: File = CoursierDependencyResolution.defaultCacheLocation
+  def defaultCacheLocation: File =
+    sys.props.get("sbt.coursier.home") match {
```

Comment:
What is the expected behavior if the user passes in a symlink?

Reply:
I'm guessing that it should use the full path of symlink without following?